docs/src/formulation-details.md

```diff
@@ -1,46 +1,64 @@
-# Three-phase formulation details
+# Formulation details
 
+```@meta
+CurrentModule = ThreePhasePowerModels
+```
 
-## `StandardACPForm`
-Real-valued formulation from:
-- Formulation without shunts: Mahdad, B., Bouktir, T., & Srairi, K. (2006). A three-phase power flow modelization: a tool for optimal location and control of FACTS devices in unbalanced power systems. In IEEE Industrial Electronics IECON (pp. 2238–2243).
+## `ACPPowerModel`
+Real-valued formulation from (extended with shunts):
+```
+@INPROCEEDINGS{4153495,
+author={B. Mahdad and T. Bouktir and K. Srairi},
+booktitle={IECON 2006 - 32nd Annual Conference on IEEE Industrial Electronics},
+title={A Three-Phase Power Flow Modelization: A Tool for Optimal Location and Control of FACTS Devices in Unbalanced Power Systems},
+year={2006},
+pages={2238-2243},
+doi={10.1109/IECON.2006.347766},
+ISSN={1553-572X},
+month={Nov},}
+```
 
 
+## `DCPPowerModel`
+This formulation applies all of the standard DC linearization tricks to the `StandardACPForm`:
+- Taylor expansion of sine: $\sin(x) \approx x$
+- Approximation of cosine: $\cos(x) \approx 1$
+- Voltage magnitude approximated as one pu: $|V_{i,a}| \approx |V_{i,b}| \approx |V_{i,c}| \approx 1$pu.
 
-## `StandardDCPForm`
-Applying all of the standard DC linearization tricks to the `StandardACPForm`
+## `SOCWRPowerModel`
+This formulation applies the standard BIM voltage cross-product (sine and cosine) substitution tricks to the `StandardACPForm`, which results immediately in a SOC formulation.
 
-## `SOCWRForm`
-Applying the standard BIM voltage cross-product (sine and cosine) substitution tricks to `StandardACPForm` results immediately in a SOC formulation.
 
-## `SDPUBFForm`
-The BFM SDP relaxation as described in:
-- Gan, L., & Low, S. H. (2014). Convex relaxations and linear approximation for optimal power flow in multiphase radial networks. In PSSC (pp. 1–9). Wroclaw, Poland. https://doi.org/10.1109/PSCC.2014.7038399
-Note that this formulation is complex-valued and additional steps are needed to implement this in JuMP.
+## `SDPUBFPowerModel`
+```@docs
+SDPUBFPowerModel
```

Comment:
@ccoffrin I don't really understand why this happens:
```
 !! Duplicate docs found for 'SDPUBFPowerModel'. [src/formulation-details.md]
```
It may have something to do with a const and the constructor having the same function name, but I don't understand why it causes problems here but not in PowerModels proper.

Reply:
Just tried these two qualified variants:
```
ThreePhasePowerModels.SDPUBFPowerModel
ThreePhasePowerModels.SDPUBFPowerModel()
``` 
Neither solves the issue. 😕

Reply:
At this point I think we need to post an issue on Documentor.jl or a question on discourse.